Add SBCL and ASDF version info to README

HM0880 opened the pull request:
The lowest-supported versions should also go in the website docs, but I do not know where those files are.

vindarel commented:
FWIW I had to upgrade ASDF to version 3.3.5 for a CI on another project, here's a one-liner:

    ( cd ~/common-lisp/ && wget https://asdf.common-lisp.dev/archives/asdf-3.3.5.tar.gz  && tar -xvf asdf-3.3.5.tar.gz && mv asdf-3.3.5 asdf )

Symbolics commented on README.md:
````diff
@@ -170,7 +175,7 @@ Create a data frame from a file named `sg-weather.csv` on the local disk:
 ```lisp
 (defparameter *df*
-	(read-csv #P"LS:DATASETS;sg-weather.csv"))
+  (read-csv #P"LS:DATASETS;sg-weather.csv"))
````
I just noted a problem here. It should now be #P"LS:DATA;sg-weather.csv". Could you please update this line as well?

Symbolics reviewed (changes requested):
I just added a comment on line 186 to address a change made in standard directory structure.  Other than that it looks good.  It would be nice if we could incorporate vindral's one-line upgrade into the README for folks that will need it.

snunez1 commented:
@HM0880, do you need help with the CLA? I'd like to merge this before I push up some changes.

HM0880 commented:
@snunez1 Just signed the CLA; sorry for the delay!

HM0880 commented:
@Symbolics @snunez1 

Just updated the data path (https://github.com/HM0880/lisp-stat/commit/6c52d4d6641675d3c604b3f46055ed90306c2067) and added the one-liner (https://github.com/HM0880/lisp-stat/commit/b4da5feb1ca3c00af580ea3a8f2faad29610272f).  Thanks for your patience.

snunez1 commented:
My apologies for taking a while on this.  It seems that the ecosystem has fixed the problem that you encountered, so the fixes no longer need to be documented.  I much appreciate the contribution.
